jobs/check_akismet_user.rb

```diff
@@ -0,0 +1,14 @@
+module Jobs
+  class CheckAkismetUser < Jobs::Base
+
+    def execute(args)
+      return if args[:user_id].blank?
+
+      user = User.where(id: args[:user_id]).first
+
+      return if user.blank?
```

Comment:
I think you meant `nil`?

```suggestion
      return if user.nil?
```